login-workflow/src/new-architecture/screens/AccountDetailsScreen/AccountDetailsScreen.tsx

```diff
@@ -13,10 +13,9 @@ export const AccountDetailsScreen: React.FC<AccountDetailsScreenProps> = (props)
     const [isLoading, setIsLoading] = useState(false);
 
     const onNext = useCallback(async (): Promise<void> => {
-        setIsLoading(true);
         try {
+            setIsLoading(true);
```

Comment:
I believe you also need to setIsLoading(false) in the catch statement, otherwise it will get stuck in the loading state if an error is thrown.